ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml

```diff
@@ -3941,7 +3957,7 @@
 	<ns1:Requirement>
 		<ns1:REQ_ID>MS-OXWSCORE_R1065</ns1:REQ_ID>
 		<ns1:Doc_Sect>2.2.4.31</ns1:Doc_Sect>
-		<ns1:Description>[In t:NonEmptyArrayOfBaseItemIdsType Complex Type] [The element "RecurringMasterItemIdRanges" with type "RecurringMasterItemIdRangesType (section 2.2.4.18)"] Specifies a union of all occurrences of a recurring master, which fit into at least one of the specified ranges.&lt;80&gt;</ns1:Description>

```

Comment:
It should be derived RS to verify like MS-OXWSCORE_R1730. Please add New RS and Comments.